lib/sync/discovery.js

```diff
@@ -1,39 +1,44 @@
 const { Subscription } = require('../models');
 const { getRepositorySummary } = require('./jobs');
 const enhanceOctokit = require('../config/enhance-octokit');
+const { getLog } = require('../../lib/config/logger');
+const probotApp = require('../worker/app');
 
 const jobOpts = {
   removeOnComplete: true,
   removeOnFail: true,
-  attempts: 3,
+  attempts: 3,   
 };
 
-module.exports.discovery = (app, queues) => async function discoverContent(job) {
+let logger = getLog();
+
+module.exports.discovery = (queues) => async function discoverContent(job) {
   const startTime = new Date();
   const { jiraHost, installationId } = job.data;
+  const app =  await probotApp(jiraHost);
   const github = await app.auth(installationId);
   enhanceOctokit(github, app.log);
 
   const repositories = await github.paginate(github.apps.listReposAccessibleToInstallation.endpoint.merge(({ per_page: 100 }), res => res.data.repositories));
   app.log(`${repositories.length} Repositories found for installationId=${installationId}`);
 
-  const subscription = await Subscription.getSingleInstallation(jiraHost, installationId);
-  if (repositories.length === 0) {
-    subscription.syncStatus = 'COMPLETE';
-    await subscription.save();
-    return;
-  }
+    const subscription = await Subscription.getSingleInstallation(jiraHost, installationId);
```

Comment:
run `npm run lint` locally and fix linting issues. there are many white spaces added